signature/src/signatures/sigstore.rs

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::image;
+use anyhow::{anyhow, Result};
+use oci_distribution::Reference;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::ffi::OsStr;
+use std::fs;
+
+// FIXME: Now, for Demo, the dir which the sigstore config files locate
+// is hardcoded to "/etc/containers/registry.d".
+const SIGSTORE_CONFIG_DIR: &str = "/etc/containers/registry.d";
```

Comment:
> Going forward I guess it is best to check if `$HOME/.config/containers/registries.d` (or `/usr/local/etc/containers/registries.d` on mac) exists before falling back to `/etc/containers/registries.d`?

Yes, this will be the follow-up work :)